static/src/javascripts/projects/common/modules/experiments/tests/integrate-smart.ts

```diff
@@ -0,0 +1,21 @@
+import type { ABTest } from '@guardian/ab-core';
+import { noop } from '../../../../../lib/noop';
+
+export const integrateSmart: ABTest = {
+	id: 'IntegrateSmart',
+	author: 'Fred OBrien (@frederickobrien)',
+	start: '2021-11-23',
+	expiry: '2022-01-10',
+	audience: 2 / 100,
+	audienceOffset: 98 / 100,
```

Comment:
Avoiding overlap with `integrateCriteo` whose audience is from 96-98%
Thanks @chrislomaxjones